Assets/Scripts/TutorialScript/Tutorial.cs

```diff
@@ -0,0 +1,21 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Tutorial : MonoBehaviour
+{
+    [SerializeField]
+    private GameObject _TutorialWin;
+    [SerializeField]
+    private GameObject _TutorialWeb;
+    private void Start()
+    {
+        #if UNITY_WEBGL
+            _TutorialWeb.SetActive(true);
+        #endif
+
+        #if UNITY_STANDALONE_WIN
+            _TutorialWin.SetActive(true);
+        #endif
```

Comment:
```suggestion
        #if UNITY_WEBGL
            _TutorialWeb.SetActive(true);
        #elif UNITY_STANDALONE_WIN
            _TutorialWin.SetActive(true);
        #endif
```
聞きかじりのまま進めるのではなく、ちゃんと体系的に調べて覚えましょう
https://learn.microsoft.com/ja-jp/dotnet/csharp/language-reference/preprocessor-directives

Reply:
すみません。上記の修正内容は
#if  
#endif
で閉じるのではなく
#if  
#elif
上記に修正します。
修正しなかった理由は、ここのコメントに
目を通してなく勘違いしてました。